TACOCAT/src/Geometry_Value.py

```diff
@@ -0,0 +1,47 @@
+#Trevor Franklin
+#Geometry Values
+
+import TACOCAT.src.SquareDhCal
+import TACOCAT.src.HexDhCal
+import numpy as np
+
+#---------------------------------------------------------------------------------------#
+# Geometry - Core
+Ar = 33.8/100 #Active Radius of the core - m
+Ac = (2*(Ar**2)*np.pi/(3*3**0.5))**0.5 #Length of Hexagonal Size - m
+
```

Comment:
Should this be here or in the read in file? We want to address this in a future issue/code planning meeting.